ocaml/xapi/xapi_bond.ml

```diff
@@ -333,19 +333,19 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
       (* 5. Members must not be the management interface if HA is enabled *)
       (* 6. Members must be PIFs that are managed by xapi *)
       (* 7. Members must have the same PIF properties *)
-      (* 8. Only the primary PIF should have a non-None IP configuration *)
+      (* 8. Only the primary PIF should have a valid IP configuration *)
```

Comment:
The word "should" isn't right: it's fine for a bond to have no IP at all. "Valid IP configuration" is vague as well, because "none" is also valid.

Better is: "At most one member may have an IP address".